### ISSUE TITLE
Date format not adapting to language/region

### ISSUE THREAD

**Issue by Altonss** [skyjake/lagrange#406]
![image](https://user-images.githubusercontent.com/66519591/143891431-0cb13885-8471-4d36-8ade-3ef82c59f883.png)
The date and time format is here not adapted to my locale (french).

On the side bar, the month is not translated:
![image](https://user-images.githubusercontent.com/66519591/143892019-a3a26071-748b-4aba-b555-0342c10c7d9f.png)
Can this string be localised?

**Comment by Altonss** [comment]
I found some string on weblate, but I don't know how to translate, it says there is an error...
https://weblate.skyjake.fi/translate/lagrange/ui/fr/?q=has%3Adismissed-check&offset=2

**Comment by skyjake** [comment]
> I found some string on weblate

That one is unrelated to these. At least the first one looks like it needs a new string added to the .po files.

**Comment by Altonss** [comment]
> That one is unrelated to these.

Yes it is unrelated, I just mentioned it because it is related to date format and I wanted to debug it xD
